packages/react-context-selector/README.md

```diff
@@ -52,12 +52,17 @@ const Counter1 = () => {
   const count1 = useCounterContext(context => context.count1);
   const increment = useCounterContext(context => context.incrementCount1);
 
+  // An easy way to test if a context provider is wrapped around this component
+  // since it's more complicated to compare with a default context value
+  const isWrappedWithContext = useHasParentContext(CounterContext);
```

Comment:
This variable is not anywhere, is it intentional? 🤔

Reply:
I guess there needs to be a better example for this